src/bottom-navigation-bar/angular/nativescript-bottom-navigation-bar.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { registerElement } from 'nativescript-angular/element-registry';
+
+import { DIRECTIVES } from './nativescript-bottom-navigation-bar.directives';
+
+@NgModule({
+    declarations: [DIRECTIVES],
+    exports: [DIRECTIVES],
+})
+export class NativeScriptMaterialBottomNavigationBarModule {}
+
+registerElement('BottomNavigationBar', () => require('../').BottomNavigationBar);
+registerElement('BottomNavigationTab', () => require('../').BottomNavigationTab);
```

Comment:
Again is not over complicated to separate all this in multiple files? It would make sense if the requires are optional. But you require them all anyway. I think a simple index.ts without a package.json is enough

Reply:
yes, right now the implementations are basics for angular, and the reason not to all in one file is only a matter of contexts, but for some cases like this #45 some improvements must be done that that's why I kept it in separate files, for those cases were it needs a more complete integration with the framework